src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -546,6 +554,76 @@ export const exchangeStitchingEnvironment = ({
             return offerResult
           },
         },
+        submitOrderOffer: {
```

Comment:
I'd plus one saying `offerOrder` (rather than `orderOffer`) since it reads better to me order is the parent model and the actual thing we're submitting.

I'd also suggest a way to make it clear that this mutation will submit the offer order _and_ create an associated inquiry/conversation. Don't have a good suggestion about naming (maybe `submitConversationalOfferOrder` or just `submitOfferOrderAndCreateConversation`?) but at least we can include a description or comment.

Reply:
Makes sense to me, going to rename it better to avoid confusion and provide a description, I missed that one

Reply:
I confess I preferred `submitOfferOrderWithConversation` with a description simple like `Submits an OfferOrder and creates a conversation for it`. But if stills confusing I can change it again